### PR TITLE
pysimdjson: 5.0.2

### PR DESCRIPTION
**Pull request by samrose**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -8768,6 +8768,8 @@ self: super: with self; {
 
   pysigset = callPackage ../development/python-modules/pysigset { };
 
+  pysimdjson = callPackage ../development/python-modules/pysimdjson { };
+
```
Comment (wegank) [inline pkgs/top-level/python-packages.nix:8772]:
```suggestion
 
```

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -8768,6 +8768,8 @@ self: super: with self; {
 
   pysigset = callPackage ../development/python-modules/pysigset { };
 
+  pysimdjson = callPackage ../development/python-modules/pysimdjson { };
+ 
   pysimplegui = callPackage ../development/python-modules/pysimplegui { };
```
Comment (wegank) [inline pkgs/top-level/python-packages.nix:8773]:
```suggestion
  pysimdjson = callPackage ../development/python-modules/pysimdjson { };

  pysimplegui = callPackage ../development/python-modules/pysimplegui { };
```

Comment (wegank) [reply pkgs/top-level/python-packages.nix:8773]: Sorry, still a space here. Also, please squash the commits and modify the message according to CONTRIBUTING.md.

File: pkgs/development/python-modules/pysimdjson/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  buildInputs = [
+    simdjson
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "pysimdjson";
```
Comment (fabaff) [inline pkgs/development/python-modules/pysimdjson/default.nix:26]:
```suggestion
    description = "Bindings for the simdjson project";
```

**Comment by fabaff** [comment]
There should be one commit in this PR: `python310Packages.pysimdjson: init at 5.0.2`

See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md

File: pkgs/development/python-modules/pysimdjson/default.nix [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  simdjson,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "pysimdjson";
+  version = "5.0.2";
+
+  src = fetchFromGitHub {
+    owner = "TkTech";
+    repo = pname;
+    rev = "6b3e58dca8f2c5505e4e90d5083e05c356809022";
```
[the remark this answers is not in the document]
Comment (fabaff) [reply pkgs/development/python-modules/pysimdjson/default.nix:15]: They are tagging the source. My bad. 

https://github.com/TkTech/pysimdjson/releases/tag/v5.0.2

**Review by MaxwellDupre — CHANGES_REQUESTED** [review]
Need to resolve conflicts and if possible squash commits to 1.

**Pull request closed without merging**